src/_main_/utils/common.py

```diff
@@ -120,24 +120,33 @@ def rename_fields(args, pairs):
 
 def serialize_all(data, full=False, **kwargs):
   #medium = (kwargs or {}).get("medium", False)
+  info = (kwargs or {}).get("info", False)
   if not data:
     return []
 
   if isinstance(data[0], dict):
     return data
 
+
   if full:
     return [d.full_json() for d in data]
+  elif info:
+    return [d.info() for d in data]
```

Comment:
Not really. We could if we want, but this part isnt something that runs automatically. The "info" field needs to be available for it to run, and its is a field that a developer would need to pass intentionally while they are developing a feature. So then it will always be available, otherwise the error will be thrown in development for the developer to catch first.